docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,221 @@
+        - determine the UEF based on the draw pattern for VERY_SMALL: `if draw_pattern == "VERY_SMALL": expected_uef = 0.8808 – (0.0008 × storage_volume)`
+        - determine the UEF based on the draw pattern for LOW: `if draw_pattern == "LOW": expected_uef = 0.9254 – (0.0003 × storage_volume)`
+        - determine the UEF based on the draw pattern for MEDIUM: `if draw_pattern == "MEDIUM": expected_uef = 0.9307 – (0.0002 × storage_volume)`
+        - determine the UEF based on the draw pattern for HIGH: `if draw_pattern == "HIGH": expected_uef = 0.9349 – (0.0001 × storage_volume)`
+        - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL
+          - check if the actual uniform_energy_factor is >= to the expected: `if uniform_energy_factor >= expected_UEF:`
+            - set rule_status to "PASS": `rule_status = "PASS"`
+          - otherwise, set rule status to FAIL and provide the note that the expected UEF is less than the required UEF: `else: rule_status = "FAIL"; rule_note = "The modeled Uniform energy Factor (UEF - " + uniform_energy_factor + ") is less than required (" + expected_UEF + ")"`
+        - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "Uniform Energy Factor is required but not provided"`
+      - else, if the storage tank is greater than 55 and less than 100 gallons, Appendix F provides a heat pump efficiency, which may be an error.  return UNDETERMINED: `elif storage_volume <=100 and storage_volume > 55:`
+        - set rule_status to UNDETERMINED: `rule_status = "UNDETERMINED"`
+        - include a note: `rule_note = "The storage tank is between 55 and 100 gallons with a capacity <= 12kW (40945 btu/hr).  Appendix F provides a heat pump efficiency for water heaters in this capacity range, but the RCT does not have a way to verify this efficiency."
+      - else if the storage volume is between 2 and 20 gallons, in which case there is no rating in APPENDIX F.  set rule status to UNDETERMINED: `elif storage_volume < 20:`
+        - set the rule status to UNDETERMINED: `rule_status = "UNDETERMINED"`
+        - add a note: `rule_note = "The storage tank is between 2 and 20 gallons.  Appendix F doesn't provide an efficiency for this water heaters of this size."
+    - otherwise, continue with rule logic: `else:`
+      - check electric storage water heaters with capacity greater than 12 kW: `if swh_type == "ELECTRIC_RESISTANCE_STORAGE" && input_capacity > 40945.7:`
+        - check that the capacity_per_volume < 4000 btu/hr/gallon: `if capacity_per_volume < 4000:`
+          - calculate the standby loss target: `standby_loss_target = 0.3 + (27/swh_eq.distribution_system.tank.storage_volume)`
+          - 
+          - if the standby_loss is less than or equal to the target, set the rule_status to PASS: `if standby_loss <= standby_loss_target: rule_status: "PASS"`
```

Comment:
I think it should only pass with an equals since App G says "and the equipment shall **match** the minimum efficiency requirements in Section 7.4.2."